cookbook/networking/fetch-data.md

```diff
@@ -0,0 +1,282 @@
+---
+layout: page
+title: "Fetch data from the internet"
+---
+
+Fetching data from the internet is necessary for most apps. Luckily, Dart and Flutter provide convenient tools to work with data from the internet!
+
+## Ingredients
+  
+  * [`http`](https://pub.dartlang.org/packages/http) package
+  * `convert` package (part of the Dart SDK)
+  * [`FutureBuilder`](https://docs.flutter.io/flutter/widgets/FutureBuilder-class.html) Widget
+  
+## Directions
+
+  1. Make a network request using the `http` package
+  2. Convert the response into a custom Dart object
+  3. Display the data
+  
+## 1. Make a network request
+
+The Dart Team publishes the [`http`](https://pub.dartlang.org/packages/http) package for making http requests. It provides the simplest way to fetch data from the internet.
+
+In this example, we'll fetch sample post from the [JSONPlaceholder REST API](https://jsonplaceholder.typicode.com/) using the [`http.get`](https://www.dartdocs.org/documentation/http/latest/http/get.html) method.
+
+```dart
+import 'package:http/http.dart' as http;
```

Comment:
do we have to prefix? makes the code noisey

Reply:
I'd prefer to keep the prefix so you can simply copy / paste the whole example and it will work, but I don't feel strongly.

Reply:
I've removed the imports from the snippets, and included them in the "Complete Example"